conda/base/context.py

```diff
@@ -869,6 +869,15 @@ def custom_multichannels(self):
             default_channels = list(self._default_channels)
 
         if self.restore_free_channel:
+            deprecated.topic(
+                "24.9",
+                "25.3",
+                topic="Adding the 'free' channel as it existed prior to conda 4.7.",
+                addendum="Add 'https://repo.anaconda.com/pkgs/free' to your channels "
+                "list manually after 'defaults', if you still need content from "
+                "the 'free' channel.",
```

Comment:
Not sure if this is the way 🤔 Back then strict channel priority was not so popular so free was source for some other versions... This would work with flexible priority, but for strict priority maybe the recommendation should be to add free in next to the other repo.anaconda.com channels under the `default_channels` multichannel.

Reply:
Do you have a suggested alternative?

Reply:
How about this?

```suggestion
                addendum="Add `https://repo.anaconda.com/pkgs/free` next to the "
                "other repo.anaconda.com hosted channel in the `default_channels` "
                "configuration option, if you still need content from the `free` "
                "channel.",
```

Reply:
I'd in favor of adding this info to the free channel docs page and referencing that:

```suggestion
                addendum="See "
                "https://docs.conda.io/projects/conda/en/stable/user-guide/configuration/free-channel.html "
                "for more details.",
```

Reply:
I've added bf903e9c5d290a145f93497b784c9216438d459b to handle this, does that suffice?